Source/WebCore/inspector/InspectorOverlay.cpp

```diff
@@ -432,15 +432,31 @@ void InspectorOverlay::paint(GraphicsContext& context)
     }
 
     for (const InspectorOverlay::Grid& gridOverlay : m_activeGridOverlays) {
+        if (m_specialGridOverlay && gridOverlay.gridNode == m_specialGridOverlay->gridNode)
```

Comment:
Instead of `m_specialXxxOverlay` can we use a name like `m_highlightedElementGridOverlay` or something else a bit more descriptive to make it easier to understand _why_ it is special?